packages/client/src/recentlyViewed/types/getRecentlyViewedProducts.types.ts

```diff
@@ -0,0 +1,12 @@
+import type { Config } from '../../types';
+import type { RecentlyViewedProducts } from './recentlyViewedProducts.types';
+
+export type GetRecentlyViewedProductsQuery = {
+  page: number;
+  pageSize: number;
```

Comment:
Because you set default values for each, I think it's better to type them.
Try this but I'm not sure if this is the way.

```suggestion
  page: number = 1;
  pageSize: number = 10;
```

Reply:
In this case, these parameters are required. What is being done is if the query parameter is not sent to the getRecentlyViewedProducts client, a default is assumed that contains both the page and pageSize parameters set to a default value.  

```javascript
const getRecentlyViewedProducts: GetRecentlyViewedProducts = (
  query = { page: 1, pageSize: 10 },
  config,
)
```
And the type `GetRecentlyViewedProducts` reflects that as query is optional: 

```javascript
export type GetRecentlyViewedProducts = (
  query?: GetRecentlyViewedProductsQuery,
  config?: Config,
) => Promise<RecentlyViewedProducts>;
```